apps/vr-tests-web-components/.storybook/preview.mjs

```diff
@@ -1,3 +1,5 @@
+import { WCThemeDecorator } from '../src/utilities/WCThemeDecorator'
```

Comment:
## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28441&commitId=e76b5076697920c88d1bca77b5ca511889a3a48e&env=prod&clientType=FLUENTUI)


  <details><summary><b>Accordion</b> 15 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Accordion.Accordion With Custom Icons.normal.chromium.png|1447| Changed|
|Accordion.Accordion With Custom Icons - Dark Mode.normal.chromium.png|0| Added|
|Accordion.Accordion With Custom Icons - RTL.normal.chromium.png|0| Added|
|Accordion.Disabled - Dark Mode.normal.chromium.png|0| Added|
|Accordion.Disabled - RTL.normal.chromium.png|0| Added|
|Accordion.Disabled.normal.chromium.png|0| Added|
|Accordion.Expand Icon Position End - Dark Mode.normal.chromium.png|0| Added|
|Accordion.Expand Icon Position End - RTL.normal.chromium.png|0| Added|
|Accordion.Expand Icon Position End.normal.chromium.png|0| Added|
|Accordion.Size - Dark Mode.normal.chromium.png|0| Added|
|Accordion.Size - RTL.normal.chromium.png|0| Added|
|Accordion.Size.normal.chromium.png|0| Added|
|Accordion.Accordion With Custom Icons RTL.normal.chromium.png|0| Removed|
|Accordion.Accordion With Custom Icons RTL.opened.chromium.png|0| Removed|
|Accordion.Accordion With Custom Icons.opened.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>